app/src/main/java/com/devindi/wallpaper/model/storage/DbModule.kt

```diff
@@ -0,0 +1,12 @@
+package com.devindi.wallpaper.model.storage
+
+import android.arch.persistence.room.Room
+import com.devindi.wallpaper.model.history.HistoryManager
+import org.koin.android.ext.koin.androidContext
+import org.koin.dsl.module.module
+
+val dbModule = module {
+    single { Room.databaseBuilder(androidContext(), MapwallDatabase::class.java, "mapwall.db").allowMainThreadQueries().build() }
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/devindi/mapwall/blob/c641b62fa39c60acdf91edee9483647734123681/app/src/main/java/com/devindi/wallpaper/model/storage/DbModule.kt#L9"></span>Exceeded max line length (100) (cannot be auto-corrected)</td>
    </tr>
  </tbody>
</table>